src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -408,7 +408,7 @@ def load_arguments(self, _):
         c.argument('enable_pod_security_policy', action='store_true', deprecate_info=c.deprecate(target='--enable-pod-security-policy', hide=True))
         c.argument('enable_pod_identity', action='store_true')
         c.argument('enable_pod_identity_with_kubenet', action='store_true')
-        c.argument('enable_workload_identity', arg_type=get_three_state_flag(), is_preview=True)
+        c.argument('enable_workload_identity', action='store_true', is_preview=True)
```

Comment:
Modifying parameter types may cause a breaking change, such as if `--enable-workload-identity True` was used in the automation script, it may no longer work

Reply:
Yes, and in our stable cli, we already used `enable` + `diable`, though we will remove the preview extension in future, we still want to align it with stable version.